### DISCUSSION TITLE
Genome Assembly

### DISCUSSION THREAD

**Discussion by CalebOA** [Kinggerm/GetOrganelle#316] [General]
Please how do I fix this "ERROR: bowtie2 not accessible"

**Comment by JianjunJin** [comment]
Please provide more information, e.g. system, environment, GetOrganelle installation approach

**Reply by CalebOA** [reply]
Thank you for your response. I was only able to figure it out just two days ago. Since I couldn't install it in my Xshell, I tried using MobaXterm and it was successful.
Sorry, I did not close this query.
